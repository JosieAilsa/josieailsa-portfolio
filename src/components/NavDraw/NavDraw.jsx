import "./NavDraw.scss";
const NavDraw = () => {
  return (
    <div className="nav__draw-wrap">
      <ul className="nav__draw">
        <li className="nav__draw-link"><a href="#home">home</a></li>
        <li className="nav__draw-link"><a href="#projects">projects</a></li>
        <li className="nav__draw-link"><a href="">skills</a></li>
        <li className="nav__draw-link"><a href="">contact</a></li>
      </ul>
    </div>
  );
};

export default NavDraw;


