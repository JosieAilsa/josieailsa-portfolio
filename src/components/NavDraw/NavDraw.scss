@use "../../assets/sass/abstracts/variables" as * ;
@use "../../assets/sass/base/resets" as * ;

.nav__draw{
    gap:2rem;
    background-color: $color-background;
    color: $color-green;
    list-style-type: none;
    display: flex;
    flex-direction: column;
    width:33vw;
    z-index: 1;
    left:70vw;
    top: 8vh;
    padding: 2rem;
    position: fixed;
    border-bottom-left-radius:1rem;

    .nav__draw-link{
        size: 2rem;
    }
}