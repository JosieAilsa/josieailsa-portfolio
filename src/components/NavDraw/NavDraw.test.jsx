import { customRender } from "../../utils/testutils";
import NavDraw from "./NavDraw";

test("Nav Draw page should render", () => {
  const { container } = customRender(<NavDraw />);
  expect(container).toMatchSnapshot();
});
