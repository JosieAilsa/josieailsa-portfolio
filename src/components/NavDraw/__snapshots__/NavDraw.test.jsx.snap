// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav Draw page should render 1`] = `
<div>
  <ul
    class="nav__draw"
  >
    <li
      class="nav__draw-link"
    >
      projects
    </li>
    <li
      class="nav__draw-link"
    >
      skills
    </li>
    <li
      class="nav__draw-link"
    >
      contact
    </li>
  </ul>
</div>
`;
